mne/time_frequency/psd.py

```diff
@@ -170,11 +375,7 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
 
     data = epochs.get_data()[:, picks][..., time_mask]
     if proj:
-        proj, _ = make_projector_info(epochs.info)
-        if picks is not None:
-            data = np.dot(proj[picks][:, picks], data)
-        else:
-            data = np.dot(proj, data)
+        epochs = epochs.apply_proj()
```

Comment:
yes it works in place, and as with `raw`, that's why the old method (with the same bugfix) should be used here